src/components/navbarbutton.tsx

```diff
@@ -0,0 +1,20 @@
+import React,{Fragment, useState} from 'react'
+import Select, { Option } from 'react-button-navbar';
+ 
+
+
+    function NavbarButton(){
+        const [current, setCurrent] = useState("opt1")
```

Comment:
This doesn't look like it's being used anywhere

Reply:
I deleted it. Thank you ☺️